lms/templates/learner_dashboard/program_details_view.underscore

```diff
@@ -1,5 +1,5 @@
 <header class="js-program-header program-header full-width-banner"></header>
-<section class="program-details-content">
+<article class="program-details-content">
```

Comment:
I don't think this is an article, [this answer](https://stackoverflow.com/questions/7549561/section-vs-article-html5) outlines the difference pretty well, I don't think this is self contained/reusable enough to be an article. I guess this is fine for now to avoid the dreaded yak shave, but a ticket+comment to address this properly would be good.

Reply:
I'm persuadable on this front. I don't think the original element was properly a section, though. What I've read suggests that <section> doesn't make much sense when it's the only section on a page. Which is how we were using it. We were almost using it like <main> here (although there's already a broader <main> on this page).

I think article is *closer* to "right" than section, though I'd believe that it's not the most right.

Reply:
To capture our offline talk -- I'll add a comment here for future people to consider the issue.